config/locales/en/course/forum/topics.yml

```diff
@@ -52,3 +52,8 @@ en:
           tag: 'Unhide'
           success: "The topic was successfully unhidden."
           failure: "The topic could not be unhidden."
+        resolved:
+          message: 'This question topic has been resolved.'
+        unresolved:
+          message: 'This question topic is unresolved.'
+          tag: 'Mark any posts as the answer to resolve the question.'
```

Comment:
If they can't mark multiple posts as the answer, it should be `Mark any post ...` instead.

Reply:
Actually multiple posts can be marked, but they can only mark one at a time. In that case which one do you think is better?

Reply:
How about `Mark helpful posts as answers to resolve this question`?

Reply:
OK 👍  will do the change.